src/ProjectTemplates/test/Templates.Blazor.WebAssembly.Auth.Tests/BlazorWasmTemplateAuthTest.cs

```diff
@@ -65,6 +65,7 @@ public Task BlazorWasmHostedTemplate_IndividualAuth_NoHttps_Works_WithLocalDB_Pr
 
     [ConditionalFact]
     [SkipOnHelix("https://github.com/dotnet/aspnetcore/issues/34554", Queues = "Windows.10.Arm64v8.Open")]
+    [QuarantinedTest("https://github.com/dotnet/aspnetcore/issues/47933")]
```

Comment:
Does quarantine attribute on a class work? IMO quarantine everything in this file.

cc @dotnet/aspnet-blazor-eng

Reply:
If this build passes I'm merging ;)

But will immediately put up a PR to optimise this. Builds have been giving me hell with this PR :)

Reply:
@JamesNK looks like there are tests there quarantined for other reasons already. Hoisting the attribute wouldn't give us what we want in terms of trackability. I think we'll have to keep disabling per test case unfortunately.